src/common/MetaPreview/MetaPreview.js

```diff
@@ -194,6 +209,15 @@ const MetaPreview = ({ className, compact, name, logo, background, runtime, rele
                         :
                         null
                 }
+                {
+                    <ActionButton
+                        className={styles['action-button']}
+                        icon={'ic_play'}
+                        label={'Show'}
+                        tabIndex={compact ? -1 : 0}
+                        href={showHref}
+                    />
+                }
                 {
                     linksGroups.has(CONSTANTS.SHARE_LINK_CATEGORY) ?
```

Comment:
the discover ui doesnt look good when we have 4 buttons here. add a check for `!compact` here so that share button is allways hidden in discover